manifest.yaml

```diff
@@ -4,6 +4,11 @@ repository: git@github.com:lyft/beam.git
 credentials: true
 slack: '#noop'
 team_email: 'no-reply@lyft.com'
+builder:
+  name: s2i
+  params:
+    builder: lyft/s2ipythonlibrary:66d815e6ba03c9515c0c2273ecca13036015715d
```

Comment:
This isn't going to work due to file layout

Reply:
then pip command will not work because those `piptools_requirements.txt` files exists in `s2ipythonlibrary` container only.